test/remote/device_tests.js

```diff
@@ -113,9 +113,9 @@ describe('remote device', function () {
                   assert.ok(device.createdAt > 0, 'device.createdAt was set')
                   assert.equal(device.name, deviceInfo.name, 'device.name is correct')
                   assert.equal(device.type, deviceInfo.type, 'device.type is correct')
-                  assert.equal(device.pushCallback, undefined, 'device.pushCallback is undefined')
-                  assert.equal(device.pushPublicKey, undefined, 'device.pushPublicKey is undefined')
-                  assert.equal(device.pushAuthKey, undefined, 'device.pushAuthKey is undefined')
+                  assert.equal(device.pushCallback, '', 'device.pushCallback is undefined')
```

Comment:
device.pushCallback is undefined or an empty string? And should this be a strict equal if you really want it to be undefined?

Reply:
Cleaned it up! Thanks!